Makefile

```diff
@@ -15,13 +15,18 @@ NXDK_STACKSIZE = 65536
 endif
 
 UNAME_S := $(shell uname -s)
+UNAME_M := $(shell uname -m)
 ifeq ($(UNAME_S),Linux)
 LD           = lld -flavor link
 LIB          = llvm-lib
 AS           = clang
 CC           = clang
 CXX          = clang++
+ifneq ($(UNAME_S),x86_64)
```

Comment:
Did you mean to compare against `UNAME_M`?

Reply:
Yes, exactly. Typo...